dagshub/auth/tokens.py

```diff
@@ -222,10 +231,23 @@ def is_valid_token(token: Union[str, Auth, DagshubTokenABC], host: str) -> bool:
             # 500's might be ok since they're server errors, so check only for 400's
             assert not (400 <= resp.status_code <= 499)
             if resp.status_code == 200:
-                assert "login" in resp.json()
-            return True
+                user = resp.json()
+                assert "login" in user
+                assert "username" in user
+                return user
         except AssertionError:
-            return False
+            return None
+
+    @staticmethod
+    def is_valid_token(token: Union[str, Auth, DagshubTokenABC], host: str) -> bool:
+        """
```

Comment:
Great job on still leaving this! I honestly forgot it's used in a couple other places 🙃